internal-services/catalog/create-advisory-task.yaml

```diff
@@ -34,7 +34,7 @@ spec:
       description: Success if the task succeeds, the error otherwise
   steps:
     - name: create-advisory
-      image: quay.io/hacbs-release/release-utils:518c0423a3b3502c7b9ef6fd8652995bec2b4e1a
+      image: quay.io/hacbs-release/release-utils:518c0423a3b3502c7b9ef6fd8652995bec2b4e1a # needs update
```

Comment:
Why does this need an update? Just to have a new image, or is there something specific that is missing? Maybe the `git_push_with_retries` function is new? Can add a link to the related PR to the description?

Reply:
Yes, it will need the `git_push_with_retries` function

Reply:
ok, cool - I got to that second PR just after reviewing this one :)